src/scores/continuous/standard_impl.py

```diff
@@ -322,7 +322,7 @@ def multiplicative_bias(
         An xarray object with the multiplicative bias of a forecast.
 
     """
-    reduce_dims = scores.utils.gather_dimensions(
+    reduce_dims = scores.utils.gather_dimensions(  # type: ignore
```

Comment:
This seems to be ignoring an issue raised due to an incorrect type hint,
All `reduce_dims` in the other `mse, mae,...` function are of type `FlexibleDimensionTypes`, yet here and in `additive_bias`, they are set to `XarrayLike`, is this correct?

Reply:
I'm pretty sure they should be FlexibleDimensionTypes as well, thanks for the pickup.

Reply:
For various reasons, I'm going to accumulate fixes on the final MR, but I have corrected this in both places you noted it, thank you.